workers/unity/Assets/Gdk/Core/Components/WorkerEntityComponents.cs

```diff
@@ -2,20 +2,40 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    /// Component denoting a worker entity
+    /// </summary>
     public struct WorkerEntityTag : IComponentData
     {
     }
 
+    /// <summary>
+    /// Component added to the worker entity immediately after establishing a connection to a SpatialOS deployment
+    /// Removed immediately after disconnecting
+    /// </summary>
     public struct IsConnected : IComponentData
     {
     }
 
+    /// <summary>
+    /// Component added to the worker entity immediately after establishing a connection to a SpatialOS deployment
+    /// Removed at the end of the tick is was added
```

Comment:
"Removed at the end of the tick is was added" sounds odd

Reply:
s/is/it